README.md

```diff
@@ -95,3 +95,60 @@ ansible-vault decrypt 'CREDENTIAL_CONFIG.yml'
 ```bash
 ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIAL_CONFIG.yml'
 ```
+
+## Updating private credential repositories
+
+Following the creation of a new release tag on the Tower Dummy credential repository, all private repostories need to be updated to reflect the latest variables and code changes. Below is a list of steps for updating a private credentials repository.
+
+1. Pull latest changes from Tower Dummy credential repository
+
+From the root of the private credential repository, add an upstream remote pointing to the Tower Dummy credential repository
+
+```bash
+git remote add upstream git@github.com:integr8ly/tower_dummy_credentials.git
+git fetch --all -p
+```
+
+Next, pull the latest changes from the new release tag created on the Tower Dummy credential repository
+
+```bash
+git pull upstream <release-tag>
+```
+
+2. Update encrypted credential configuration file
+
+Each private credentials repository should have an associated encrypted credentials configuration file. Locate this file and decrypt using the below command:
+
+```bash
+ansible-vault decrypt $CREDENTIAL_CONFIG_FILE
+```
+
+Next, update the decrypted credentials configuration file with any new variables that have been included in the new release tag of the Tower Dummy credential repository (See [CHANGELOG](./CHANGELOG.md) for reference)
+
+Once all variables have been added, re-run the ansible bootstrap task
+
+```bash
+ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIAL_CONFIG_FILE.yml'
+```
+
+Finally, re-encrypt the updated credentials configuration file
+
+```bash
+ansible-vault encrypt $CREDENTIAL_CONFIG_FILE
+```
```

Comment:
The CREDENTIAL_CONFIG.yml file will be automatically encrypted by the bootstrap playbook.